user_guides/jakartaee/src/main/jbake/content/rules.adoc

```diff
@@ -459,12 +459,19 @@ TCK and the following Technology Compatibility Kits (TCKs):
 * Jakarta Dependency Injection 2.0 
 * Jakarta Bean Validation 3.0
 * Jakarta XML Binding 3.0 (If XML Binding is supported)
+* Jakarta JSON Processing 2.1
+* Jakarta JSON Binding 3.0
```

Comment:
Other new Standalone TCKs to consider adding:
- Jakarta RESTful Web Services 3.1
- Jakarta Concurrency 3.0  
- Jakarta Server Faces 4.0
- Jakarta Batch 2.1

Reply:
@lukasj pinging you on this comment, do you expect that these other new Standalone TCKs will be added to the Platform TCK by this pull request?

Reply:
I have no idea which technologies do have its own TCKs and which don't, so I cannot guarantee that what I'll add is correct. All I know for sure is that JSONP/B do belong to the set of TCKs removed from the platform TCK.

Reply:
that is fine, we can update after this is merged.